pkgs/development/tools/build-managers/wafHook/setup-hook.sh

```diff
@@ -21,6 +21,11 @@ wafConfigurePhase() {
     echoCmd 'configure flags' "${flagsArray[@]}"
     python "$wafPath" "${flagsArray[@]}"
 
+    if ! [[ -v enableParallelBuilding ]]; then
+        enableParallelBuilding=1
+        echo "waf: enabled parallel building"
```

Comment:
```suggestion
```
don't think we need to log that tbh.

Reply:
cmake does it, so it's fine to have here as well

Reply:
I mark this conversation as resolved. Please reopen/open a new one if you strongly disagree and think further discussion is needed.